File: src/net/t3kt/tctrl/schema/params/TriggerParamSchema.java
package net.t3kt.tctrl.schema.params;

import net.t3kt.tctrl.schema.TctrlSchemaProto.ParamSpec;

public final class TriggerParamSchema extends SingleParamSchema {
    TriggerParamSchema(ParamSpec spec) {
        super(spec);
    }

    public String getButtonText() {
        return spec.getButtonText();
    }
}

File: win/TctrlCore/Schema/Params/TriggerParamSchema.cs
using System;
using Tctrl.Schema;

namespace Tctrl.Core.Schema.Params
{
    public sealed class TriggerParamSchema : SingleParamSchema
    {
        public TriggerParamSchema(ParamSpec spec) : base(spec)
        {
        }

        public String ButtonText => Spec.ButtonText;
    }
}